lib/uiautomator2.js

```diff
@@ -266,6 +266,8 @@ class UiAutomator2Server {
       cmd.push('-e', 'DISABLE_SUPPRESS_ACCESSIBILITY_SERVICES', this.disableSuppressAccessibilityService);
     }
     cmd.push(INSTRUMENTATION_TARGET);
+    // Disable Google analytics to prevent possible fatal exception
+    cmd.push('-e', 'disableAnalytics', true);
```

Comment:
shouldn't we push it before setting the target?

Reply:
oh, that's right